V1.01_ROM_disassembly_C000.asm

```diff
@@ -391,16 +399,17 @@ label_c208:
     ld      b,e                             ;[c212] 43
     ld      hl,($ffbd)                      ;[c213] 2a bd ff
 label_c216:
-    in      a,($82)                         ;[c216] db 82
-    bit     2,a                             ;[c218] cb 57
-    jr      z,label_c216                    ;[c21a] 28 fa
-    in      a,($c0)                         ;[c21c] db c0
-    bit     5,a                             ;[c21e] cb 6f
-    jr      z,label_c229                    ;[c220] 28 07
-    outi                                    ;[c222] ed a3
-    jr      nz,label_c216                   ;[c224] 20 f0
-    dec     d                               ;[c226] 15
-    jr      nz,label_c216                   ;[c227] 20 ed
+    in      a,($82)                         ;[c216] read PORTC
+    bit     2,a                             ;[c218]
+    jr      z,label_c216                    ;[c21a]
+
+    in      a,($c0)                         ;[c21c]
+    bit     5,a                             ;[c21e]
+    jr      z,label_c229                    ;[c220]
+    outi                                    ;[c222]
+    jr      nz,label_c216                   ;[c224]
+    dec     d                               ;[c226]
+    jr      nz,label_c216                   ;[c227]
```

Comment:
This change conflicts with FDC branch. Since is only one line comment, I suggest to revert-and-squash to avoid merging issues